src/lib/Microsoft.Health.Common/Telemetry/ITelemetryMetric.cs

```diff
@@ -1,12 +1,12 @@
-﻿// -------------------------------------------------------------------------------------------------
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
-// -------------------------------------------------------------------------------------------------
-
-namespace Microsoft.Health.Common.Telemetry
-{
-    public interface ITelemetryEvent
-    {
-        string EventName { get; }
-    }
-}
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Common.Telemetry
+{
+    public interface ITelemetryMetric
```

Comment:
nit: I think the naming here is a little misleading. ITelemetryMetric seems like it would be the metric itself, but the functionality it provides is a metric getter. Looking at the implementation it appears that classes that implement this interface can be converted to a metric.

If it makes sense I'm wondering if the interface should be called 'ITelemetryFormattable', and a method 'ToMetric()'.

Reply:
I agree it was a little bit ambiguous/confusing, thank you for the feedback. Updated the interface name and method name